kernel/src/main/java/org/kframework/compile/checks/CheckRewrite.java

```diff
@@ -130,7 +131,7 @@ public void apply(KApply k) {
                 }
             }
         }.accept(body);
-        if (!h.hasRewrite) {
+        if (!h.hasRewrite && !isClaim) {
             errors.add(KEMException.compilerError("Rules must have at least one rewrite.", body));
         }
```

Comment:
Are we not enabling this check on `claim`? They should have at least one rewrite as well, I would think?

Reply:
There are several claims which don't contain rewrites:
https://github.com/kframework/k/blob/master/k-distribution/tests/regression-new/kbmc/success-test1-spec-bmc.k#L13
There a couple more but I can't find them right now.